src/Router.sol

```diff
@@ -122,21 +161,32 @@ contract Router is IRouter, EIP712, FeeVerifier {
         emit PauserSet(pauser_);
     }
 
+    /// @notice Rescue ERC-20 tokens in case of stuck tokens by owner
+    /// @param token The token address
+    /// @param receiver The receiver address
+    /// @param amount The amount of tokens to be rescued
     function rescue(address token, address receiver, uint256 amount) external onlyOwner {
         IERC20(token).safeTransfer(receiver, amount);
     }
 
+    /// @notice Pause `execute` and `executeWithSignature` by pauser
     function pause() external onlyPauser {
         paused = true;
         emit Paused();
     }
 
+    /// @notice Resume `execute` and `executeWithSignature` by pauser
     function resume() external onlyPauser {
         paused = false;
         emit Resumed();
     }
 
-    /// @notice Execute logics through current user's agent. Create agent for user if not created.
+    /// @notice Execute arbitrary logics through the current user's agent. Creates an agent for users if not created.
+    ///         Charge fees based on the scenarios defined in the FeeVerifier contract, which calculates fees on-chain.
```

Comment:
Not sure if the indent is a correct format for spec parsing.

Reply:
The parser simply encodes '\n' and indent:
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/4598616/237033619-f009cdec-5338-4e27-9316-516af2f6ba92.png">

The Solidity official documentation doesn't give a specific rule for this, and the referenced GitHub issue(https://github.com/ethereum/solidity/issues/11359) doesn't arrive at a solid conclusion either.

In our past development, we have chosen this particular style, which aligns with what is used by Curve, Gnosis, etc. This choice is also noted in this [ticket](https://app.asana.com/0/1110393615095724/1202201092359525/f).